### PR TITLE
Added MemoryValue Plugin

### PR DESCRIPTION
**Pull request by hesirui**
## Basics

Check relevant points but **please do not remove entries**.
Do not describe the purpose of this PR in the PR description but:

- [ ] Short descriptions should be in the release notes (added as entry in
      `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add something to the the release notes.**
- [ ] Longer descriptions should be in documentation or in design decisions.
- [ ] Describe details of how you changed the code in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, should be in the commit messages.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests
- [ ] I ran all tests locally and everything went fine
- [x] affected documentation is fixed
- [ ] I added code comments, logging, and assertions (see doc/CODING.md)
- [x] meta data is updated (e.g. README.md of plugins and doc/METADATA.md)

## Review

Remove the line below and add the "work in progress" label if you do
not want the PR to be reviewed:

@markus2330 please review my pull request

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for creating this PR!

File: src/plugins/memoryvalue/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+**Memory Value Plugin**
```
Comment (kodebach) [inline src/plugins/memoryvalue/README.md]: The metadata is missing. See https://github.com/ElektraInitiative/libelektra/blob/master/src/plugins/template/README.md

Also, use [markdown headings](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#headings) for the title.

**Comment by markus2330** [comment]
This pull request **introduces 1 alert** when merging 55fb85f97777218ed779ecd8158cc62be89d6973 into 54a4c0194946917b7d093e0777f465619b2f3d6f - [view on LGTM.com](https://lgtm.com/projects/g/ElektraInitiative/libelektra/rev/pr-83f51972e8d1a5664f7531a3afbf97bd60ebaf1a)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by markus2330** [comment]
Thank you for the update!

Something went wrong in the last merge (see the "Files changes" here). Please rebase to master.

Please also look at the build server output (https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2699/4/pipeline/223) you need to rebase as described in https://master.libelektra.org/doc/CODING.md

**Comment by hesirui** [comment]
I could not resolve these problems with rebasing thus I created a new PR. https://github.com/ElektraInitiative/libelektra/pull/2699

**Pull request closed without merging**